src/common/normSession.cpp

```diff
@@ -7,7 +7,7 @@
 #include <time.h>  // for gmtime() in NormTrace()
 
 #include "protoPktETH.h"
-#include "protoPktIP.h"
+#include "protoPktIp.h"
```

Comment:
I'm not sure this change is correct?  The "ProtoPktIP" header file is named with uppercase "IP" per the original code.

Reply:
Fixed